packages/shared/providers.yaml

```diff
@@ -1318,6 +1318,11 @@ linkhut:
     proxy:
         base_url: https://api.ln.ht
     docs: https://docs.nango.dev/integrations/all/linkhut
+mailgun:
+    categories:
+        - marketing
+    auth_mode: API_KEY
+    docs: https://docs.nango.dev/integrations/all/mailgun
```

Comment:
Ideally we should verify the credentials (cf. `proxy.verification` field).

Reply:
Not sure exactly how to do that.